stdlib/2/md5.pyi

```diff
@@ -1,10 +1,6 @@
 # Stubs for Python 2.7 md5 stdlib module
 
-class md5(object):
-    def update(self, arg: str) -> None: ...
-    def digest(self) -> str: ...
-    def hexdigest(self) -> str: ...
-    def copy(self) -> md5: ...
+from hashlib import _hash as md5
```

Comment:
That last part is not true but using `md5` is still more accurate; `md5.md5` is the same as `hashlib.md5` and it's a function that returns a `_hash` instance, not a class.